### ISSUE TITLE
bundletool doesn't fetch the atrributes in styles.xml if there are more than one dpi.

### ISSUE THREAD

**Issue by baien** [google/bundletool#128]
**Describe the bug**
I have an layout xml, which uses a textview, like this:

```
<TextView 
        android:id="@+id/title_text"
        android:layout_toRightOf="@id/app_icon"
        android:layout_marginLeft="10dip"
        android:layout_centerVertical="true"
        style="@style/title_text_style"
        android:text="@string/apps_locker_app_name"
        android:singleLine="true"
        android:lines="1"
        android:ellipsize="end"
        />
```

The style in the styles.xml is like this:

```
<style name="title_text_style">
     <item name="android:textSize">16sp</item>
     <item name="android:layout_width">wrap_content</item>
     <item name="android:layout_height">wrap_content</item>
</style>
```

however, I have 2 styles.xml in styles folder and styles-ldpi folder, the other style in ldpi is as follow:

```
<style name="title_text_style">
    <item name="android:textSize">20sp</item>
    <item name="android:layout_width">wrap_content</item>
    <item name="android:layout_height">wrap_content</item>
</style>
```

After using bundletool to install, the app always crash on the phone under Android 9. (Android 9 is ok) From the log it shows that I miss the layout_width in the layout xml file. If I remove the title_text_style style in the styles-ldpi, the app will launch without an error.

**Bundletool version(s) affected**
Version: [bundletool-all-0.11.0.]

**Stacktrace**
java.lang.RuntimeException: Binary XML file line #28: You must supply a layout_width attribute.

**To Reproduce**
in my case, use style to describe the layout_width in both styles folder and styles-ldpi folder, it will crash in all phones under Android 9

**Expected behavior**
No crashing.

**Environment:**
Android 8.0 and pre

**Comment by plecesne** [comment]
Do you have a small project or AAB that reproduces the issue and that you could share with us so we can get a closer look?

**Comment by baien** [comment]
It is very easy to verify. I just newed a HelloWorld project, and add a style named "title_text_style" in both styles under both folder values and values-ldpi as mentioned before. Remove the layout_width and layout_height attribute of the TextView in the HellowWorld and use the style=@style/title_text_style instead. build the abb, and install it with bundletool. The app will crash. 
[myapplication.zip](https://github.com/google/bundletool/files/3861340/myapplication.zip)

**Comment by baien** [comment]
> Do you have a small project or AAB that reproduces the issue and that you could share with us so we can get a closer look?

Hi, plecesne, is there any more information I could offer to help you look into this issue?

**Comment by plecesne** [comment]
We have all the information we need, sorry for the delay, we will be looking into it as soon as possible.

**Comment by plecesne** [comment]
Thanks to your project, I was able to reproduce, we're looking into the best way to fix the issue.

**Comment by ymakhno** [comment]
Fixed in 0.14.0.
